src/main/cpp/feedback/DriverFeedback.cpp

```diff
@@ -92,21 +98,19 @@ void DriverFeedback::UpdateLEDStates()
 
 void DriverFeedback::UpdateDiagnosticLEDs()
 {
-    /**
-    StateMgr *noteStateManager = MechanismConfigMgr::GetInstance()->GetCurrentConfig()->GetMechanism(MechanismTypes::NOTE_MANAGER);
-    auto noteMgr = noteStateManager != nullptr ? dynamic_cast<noteManagerGen *>(noteStateManager) : nullptr;
-    if (noteMgr != nullptr)
+
+    StateMgr *taleStateManager = MechanismConfigMgr::GetInstance()->GetCurrentConfig()->GetMechanism(MechanismTypes::DRAGON_TALE);
```

Comment:
You need to check if the current config is not a null pointer as well like you checked if the intake and dragon tale manager are null pointers